styles/_motebehov.less

```diff
@@ -0,0 +1,95 @@
+.motebehovInnholdLenke {
+  h3 {
```

Comment:
Prøv å flate ut CSS-en i denne filen og følge BEM-navnekonvensjon. Nå nøstes det for mye.